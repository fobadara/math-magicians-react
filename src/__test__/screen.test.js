import { render, cleanup } from '@testing-library/react';
import Screen from '../components/screen';

afterEach(cleanup);

test('should create snapshot', () => {
  const state = { total: '0' };
  const { asFragment } = render(<Screen state={state} />);
  expect(asFragment(<Screen state={state} />)).toMatchSnapshot();
});
