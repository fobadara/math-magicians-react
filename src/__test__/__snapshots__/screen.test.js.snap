// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create snapshot 1`] = `
<DocumentFragment>
  <div
    class="text-break h-25"
    id="screen"
  >
    <span>
      0
    </span>
    <span />
    <span />
  </div>
</DocumentFragment>
`;
